Timeline/src/index.ts

```diff
@@ -47,10 +47,9 @@ export default class CjaasTimelineWidget extends LitElement {
   @property({ type: String, attribute: "tape-read-token" }) tapeReadToken:
     | string
     | undefined;
-  @property({ type: String, attribute: "stream-read-token" }) streamReadToken:
+  @property({ type: String, attribute: "stream-read-token" }) streamToken:
```

Comment:
Why rename the property? Misses info on permission required.

Reply:
@venkateshwarv simplification, basically. seems to me that within the context of the widget, there's really only one permission needed

Reply:
@venkateshwarv  Actually, I don't know if it's worth fretting too much over right now . . . the next iteration is going to negate specific SAS tokens anyway since they will be generated with internal logic and more out of site from users